src/components/List_POC/Accordion.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from "react";
+// import PropTypes from "prop-types";
+
+import AccordionItem from "./AccordionItem";
+
+class Accordion extends Component {
+  // static propTypes = {
+  //   allowMultipleOpen: PropTypes.bool,
+  //   children: PropTypes.any.isRequired
+  // };
+
+  static defaultProps = {
+    allowMultipleOpen: false
+  };
+
+  constructor(props) {
+    super(props);
+
+    const openSections = {};
+
+    this.state = { openSections };
+  }
+
+  updateOpenItems = () => {
+    const updatedOpenSections = [];
+    if (React.children) {
```

Comment:
Does this reference ‘this.props.children’

Reply:
Yes, React.children provides utilities for dealing with this.props.children opaque data structure.